src/facade/redis_parser.cc

```diff
@@ -15,7 +15,7 @@ namespace {
 
 // When changing this constant, also update `test_large_cmd` test in connection_test.py.
 constexpr int kMaxArrayLen = 65536;
-constexpr int64_t kMaxBulkLen = 64 * (1ul << 20);  // 64MB.
+constexpr int64_t kMaxBulkLen = 256 * (1ul << 20);  // 256MB.
```

Comment:
Would it make sense to flagify this value?

Reply:
I think the this is a very big key.. when creating bigger keys the latency of request increases.
I have another task to expose the limitation we have but I am not sure why DF user will want to have keys size so big

Reply:
It's not just about the latency. Having ~256MB blobs also effects the memory in-flight. Basically, you can not control your memory usage since you have thousands of requests in flights at any given moment.

Reply:
You're both right, but ultimately I'd say that this it the user's decision. The could over-provision for the ephemeral memory and accept the latency penalty.
I agree that we should not _encourage_ large keys, but a flag is hidden enough to not be an encouragement IMHO.
Anyway, I seem to hold a minority opinion :)